front/app/components/admin/GraphCards/VisitorsCard/Chart/index.tsx

```diff
@@ -25,6 +25,7 @@ type Props = Dates &
   Resolution & {
     timeSeries: TimeSeries | NilOrError;
     innerRef: React.RefObject<any>;
+    hideTooltip?: boolean;
```

Comment:
Note: Need to hide tooltips else they print if your mouse happens to be over them when the print window loads